lib/resolver/go-universal.js

```diff
@@ -1,12 +1,36 @@
 import goFile from './go-file.js';
 
+function githubUrls(url) {
+  const [, user, repo, ...path] = url.split('/');
+
+  if (!path.length) {
+    return [
+      `{BASE_URL}/${user}/${repo}/blob/master/${repo}.go`,
+      `{BASE_URL}/${user}/${repo}`,
+    ];
+  }
+
+  const fullPath = path.join('/');
+  const last = path.slice(-1);
+
+  return [
+    `{BASE_URL}/${user}/${repo}/blob/master/${fullPath}/${last}.go`,
+    `{BASE_URL}/${user}/${repo}/tree/master/${fullPath}`,
+    `{BASE_URL}/${user}/${repo}`,
+  ];
+}
+
 export default function ({ target, path }) {
   const isPath = !!target.match(/^\.\.?[\\|\/]?/);
 
   if (isPath) {
     return goFile({ path, target });
   }
 
+  if (target.startsWith('github.com')) {
```

Comment:
If we ever [support GitHub Enterprise](https://github.com/OctoLinker/browser-extension/issues/75), we'll have to change this into a more general `isGithubUrl` function (similar to what happened in 71ae1d68526919bee2c6e8339a1cd87b5febff11). I think it's ok for now though, especially since it's currently the only instance of this logic. 👍